thunder/distributed/__init__.py

```diff
@@ -177,6 +177,102 @@ def prep_shard(
         )
 
 
+# When the user calls ddp(jitted_module), this function does the following
+# - Marks the original function with appropiate attributes (use_ddp...)
+# - Broadcasts parameters if necessary
+# - It then registers a transform (callback that runs before prologue is executed) that transforms the
+#   prologue and compute trace, that insert syncs (and grad syncs for the backward, handled by thunder automatically.)
+
+
+def ddp_transform_module(
+    thunder_model: ThunderModule,
+    *,
+    broadcast_from=None,
```

Comment:
could you give type hint to this keyword only argument? it feels like `int | None` though